Python/sysmodule.c

```diff
@@ -1903,20 +1900,25 @@ _PySys_GetSizeOf(PyObject *o)
     return (size_t)size + presize;
 }
 
+/*[clinic input]
+sys.getsizeof
+
+    object: object
+    default as dflt: object = NULL
```

Comment:
I suggest to use a longer name rather than a shorter one:

```suggestion
    default as default_size: object = NULL
```

Reply:
That would make the diff unnecessary bigger. IMO, it is better to align with the current naming.